src/main/java/org/team1540/robot2024/subsystems/elevator/ElevatorIOTalonFX.java

```diff
@@ -0,0 +1,99 @@
+package org.team1540.robot2024.subsystems.elevator;
+
+import org.team1540.robot2024.Constants;
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.MotionMagicConfigs;
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.ForwardLimitValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.ctre.phoenix6.signals.ReverseLimitValue;
+
+public class ElevatorIOTalonFX implements ElevatorIO {
+
+    private final MotionMagicVoltage motionMagicOut = new MotionMagicVoltage(0).withEnableFOC(true);
+    private final TalonFX leader = new TalonFX(Constants.Elevator.TALON_ID_1);
+    private final TalonFX follower = new TalonFX(Constants.Elevator.TALON_ID_2);
+
+    private final StatusSignal<Double> leaderPosition = leader.getPosition();
+    private final StatusSignal<Double> leaderVelocity = leader.getVelocity();
+    private final StatusSignal<Double> leaderAppliedVolts = leader.getMotorVoltage();
```

Comment:
For these status signals, we aren't getting the corresponding status from the follower, so the "leader" prefix should be removed for less verbosity

Reply:
I'm going to do nothing here.